lua/lean/_util.lua

```diff
@@ -12,6 +12,19 @@ function M.tbl_repeat(value, times)
   return result
 end
 
+--- Fetch the diagnostics for all Lean LSP clients from the current buffer.
+function M.lean_lsp_diagnostics(opts, bufnr)
+  opts = opts or {}
+  local diagnostics = {}
+  for _, client in pairs(vim.lsp.buf_get_clients(bufnr)) do
+    if client.name:match('^lean') then
+      opts.namespace = vim.lsp.diagnostic.get_namespace(client.id)
+      vim.list_extend(diagnostics, vim.diagnostic.get(bufnr, opts))
```

Comment:
```suggestion
      vim.list_extend(diagnostics, vim.diagnostic.get(bufnr or 0, opts))
```
I think you need an explicit first parameter for the second to be considered.

Reply:
Thanks!